src/Model/proxyHandler.ts

```diff
@@ -1,11 +1,24 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
+import { BaseModel } from './Model';
 
 export const proxyHandler: ProxyHandler<any> = {
-  get(target: any, prop: string | symbol) {
+  get(target: any, prop: string | symbol, receiver: any) {
+    const Model = target.constructor as typeof BaseModel;
+    if (Model.$hasComputed(prop as string)) {
+      const property = Object.getOwnPropertyDescriptor(
+        Object.getPrototypeOf(target),
+        prop,
+      );
+      if (property?.get) {
+        return property.get.call(target.$attributes);
+      }
```

Comment:
In Lucid, it's supposedly different. I think the getter ends up being called on the Proxy, so you are able to access attributes, but also methods on the model.

Reply:
I tried different combinaisons (with receiver argument of proxy handler), but only this one works without breaking other tests

Reply:
Ok, I retried with Reflect.get and it's working now.